docs/protocol.md

```diff
@@ -105,65 +103,172 @@ In the table below:
 
 > For keys, we use the notation $X_{A,B}$, where $X$ represents the key owner
 > ($`X \in \{NR, J, S\}`$ [for newsroom, journalist, and source, respectively]),
-> $A$ represents the key's usage ($`A \in \{pke,sig,fetch,dh\}`$), and is prefixed
-> with an "e" if the key is ephemeral. $B$ indicates whether the component is
+> $A$ represents the key's usage ($`A \in \{sig,fetch,pke,pq,md\}`$), and is prefixed
```

Comment:
We should sync this notation across:

1. this description;
2. the "Usage" column in the table, which maybe obviates (1) altogether;
3. the function signatures for `SD-PKE` and `SD-APKE`; and
4. their invocations, e.g.:
    https://github.com/freedomofpress/securedrop-protocol/blob/f0a08210ec0233c5b5dd49d6ae41476fb11a96bc/docs/protocol.md?plain=1#L353-L360